sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SparkBatchClientLiveTests.cs

```diff
@@ -47,7 +46,34 @@ public async Task TestSparkBatchJob()
             SparkBatchOperation createOperation = await client.StartCreateSparkBatchJobAsync(createParams);
             SparkBatchJob jobCreateResponse = await createOperation.WaitForCompletionAsync();
 
-            // Verify the Spark batch job completes successfully
+            // Verify the Spark batch job submission completes successfully
+            Assert.True("starting".Equals(jobCreateResponse.State, StringComparison.OrdinalIgnoreCase) || "running".Equals(jobCreateResponse.State, StringComparison.OrdinalIgnoreCase),
```

Comment:
Should there be an enum for possible values of `SparkBatchJob.State`, rather than testing strings?

Reply:
Create a PR for this requirement: https://github.com/Azure/azure-rest-api-specs/pull/16033

Reply:
In the next SDK release, we will have an enum for possible values of `SparkBatchJob.State`